packages/server/src/index.ts

```diff
@@ -628,12 +647,12 @@ export class App {
 
         // Send input message and get prediction result (External)
         this.app.post('/api/v1/prediction/:id', upload.array('files'), async (req: Request, res: Response) => {
-            await this.processPrediction(req, res, socketIO)
+            await this.processPrediction(req, res, socketIO, req.body.apiType)
         })
 
         // Send input message and get prediction result (Internal)
         this.app.post('/api/v1/internal-prediction/:id', async (req: Request, res: Response) => {
-            await this.processPrediction(req, res, socketIO, true)
+            await this.processPrediction(req, res, socketIO, 'internal')
```

Comment:
This can be enum, like ChatTypes.internal instead of hardcoding

Reply:
The only feedback I can give for mr instead of code pices, here would be to get variables from an enum, or a common constant, instead of hardcoded texts. I think it will benefit in the future.